vision/unit_tests/test__gax.py

```diff
@@ -32,6 +32,28 @@ def test_ctor(self):
             api = self._make_one(client)
         self.assertIs(api._client, client)
 
+    def test_annotation(self):
+        from google.cloud.vision.feature import Feature
+        from google.cloud.vision.feature import FeatureTypes
+        from google.cloud.vision.image import Image
+
+        client = mock.Mock()
```

Comment:
Can you add `spec` to limit the "oops" effect? You can start with

```python
client = mock.Mock(spec=[])
```

and then add strings to the `spec` list as you realize you need missing attributes